populate/src/main/resources/seed/portals/demo/portal.json

```diff
@@ -16,7 +16,7 @@
         "password": "broad_institute",
         "initialized": true,
         "emailSourceAddress": "support@juniper.terra.bio",
-        "defaultLanguage": "es"
+        "defaultLanguage": "en"
```

Comment:
For better or for worse, much of the existing code relied on the english translations. While possible to make it work with spanish on demo, I figured it would be easier to just not deal with it and change the default language over to english like we'd discussed.